pkg/collect/host_http.go

```diff
@@ -32,15 +32,15 @@ func (c *CollectHostHTTP) Collect(progressChan chan<- interface{}) (map[string][
 	case httpCollector.Get != nil:
 		response, err = doRequest(
 			"GET", httpCollector.Get.URL, httpCollector.Get.Headers,
-			"", httpCollector.Get.InsecureSkipVerify, httpCollector.Get.Timeout)
+			"", httpCollector.Get.InsecureSkipVerify, httpCollector.Get.Timeout, httpCollector.TLS.CACert, httpCollector.Proxy)
```

Comment:
You need to handle the case where `httpCollector.TLS == nil`. Tests are panicing because if this. I'd pass in `httpCollector.TLS` as the parameter then in `doRequest`  the perform the checks there

Reply:
thanks, fixed!